playwright-e2e/pages/pancan/enrollment/consent-form-page.ts

```diff
@@ -2,26 +2,44 @@ import { Page } from '@playwright/test';
 import Question from 'lib/component/Question';
 import Input from 'lib/widget/Input';
 import { PancanPage } from 'pages/pancan/pancan-page';
+import { PatientsData, TypePatient } from 'pages/patient-type';
+import { generateUserName } from 'utils/faker-utils';
+import * as user from 'data/fake-user.json';
 
 export default class ConsentFormPage extends PancanPage {
-  constructor(page: Page) {
+  constructor(page: Page, private typePatient: TypePatient = 'adult') {
```

Comment:
UI fields on `Consent Form` pages are identical for Child and Adult enrollment except some minor text label difference. `TypePatient` is used here.